src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,15 +1,23 @@
 package core.mate.academy.model;
 
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
 /**
  * Add some custom fields that could be only in Excavator
  * Do not remove no-args constructor
  */
-public class Excavator extends Machine {
+public class Excavator extends Machine implements MachineProducer<Excavator> {
     public Excavator() {
     }
 
     @Override
     public void doWork() {
         System.out.println("Excavator started to work");
     }
+
+    @Override
+    public List<Excavator> get() {
+        return List.of(new Excavator(), new Excavator());
+    }
 }
```

Comment:
no need to implement from MachineProducer
instead, you should create separate package `producer`, and create Producers for each model, implemented MachineProducer<?> interface
also, according to the task, you should add some fields to this model
![image](https://github.com/mate-academy/jv-pecs/assets/73041091/e96a6970-9bac-40b2-864e-3d1b13d1e9be)
```suggestion
public class Excavator extends Machine {
    public Excavator() {
    }

    @Override
    public void doWork() {
        System.out.println("Excavator started to work");
    }
}
```